src/lib/orionTypes/TypeEntity.cpp

```diff
@@ -116,7 +105,30 @@ std::string TypeEntity::render
     else
     {
       out += valueTag(indent  + "  ", "name", type, ciP->outFormat, true);
-      out += contextAttributeVector.render(ciP, EntityTypes, indent + "  ", false, true);
+      if (ciP->outFormat == JSON) {
+        out += indent + "  \"attributes\": [ ";
+        for (unsigned int ix = 0; ix < contextAttributeVector.size(); ix++)
```

Comment:
Well ...
The render functionality is a 'tree' where each node defers the work of rendering to its children.
The way this function worked before, ContextAttributeVector::render() rendered the vector (just doing a loop, calling the render method of each attribute in the vector).
Here you have implemented the ContextAttributeVector::render inside TypeEntity::render.
This is not a very good idea. It gives copies of the code ...

It would be better to move this entity code block to ContextAttributeVector::render(), where it belongs.
This method should not be modified at all, it should keep delegating the work to ContextAttributeVector::render().